documentation/interpretation_guide.md

```diff
@@ -103,6 +121,8 @@ Sometimes governments make ambiguous funding announcements (where it is not clea
 
 ### H2 - Testing Policy
 
+- If there is a policy which says that everyone who is symptomatic could walk in and get a test, but this could not be met if everyone symptomatic walked into the clinic at once, this suggests there is not the capacity to meet this. While we are coding government policies, capacity issues affect implementation. This therefore requires judgment, about how aspirational a policy is versus how practical it is to see if it is able to meet its propositions. Therefore as well as the policy announced, evidence should be provided demonstrating that there is capacity on the ground to meet this testing policy, mirroring the capacity logic used for H3.
+- This indicator does not have a binary flag variable to reflect geographic variation in policies. This implies that this policy applies generally across the whole jurisdiction. Therefore, we generally code the level of policy that applies everywhere across the jurisdiction- the highest common value of policy nationwide or statewide. We do not code any policies of a higher value that are only available in limited areas. *For example* if there is not testing available to anyone symptomatic across the whole country or state, but only in certain areas, this cannot be coded as a 2 as this does not apply generally. Therefore the highest common value nationwide or statewide is a 1, and this is coded. 
```

Comment:
As with others: let's get rid of second sentence. And re-word for data users, not coders.